js/exporter/jspdf/v3/export_data_grid_3.js

```diff
@@ -96,15 +97,24 @@ function exportDataGrid(doc, dataGrid, options) {
             );
 
             const docStyles = getDocumentStyles(doc);
-            drawCellsContent(doc, options.customDrawCell, pdfCellsInfo, docStyles);
-            drawCellsLines(doc, pdfCellsInfo, docStyles);
-
-            const isDrawTableBorderSpecified = options.drawTableBorder === true;
-            const isEmptyPdfCellsInfoSpecified = isDefined(pdfCellsInfo) && pdfCellsInfo.length === 0;
-            if(isDrawTableBorderSpecified || isEmptyPdfCellsInfoSpecified) {
-                const tableRect = calculateTableSize(doc, rowsInfo, options); // TODO: after splitting to pages we need get 'rowsInfo' for selected table in the page
-                drawGridLines(doc, tableRect, docStyles);
-            }
+
+            const pdfCellsInfoByPage = applySplitting(pdfCellsInfo, options);
```

Comment:
splitToPages works with 'rect' objects only.
It doesn't require other properties of 'cellInfo' objects.
Can we change splitToPages API to accept a 'rect' array as a primary argument instead of a 'cellInfo' array?
This change allow to remove all the 'cellInfo' related code from the 'splitToPages' method and rename it to 'splitRectsToPages'.

const rects = pdfCellsInfo.map(cellInfo => Object.assign({}, cellInfo.rect, {sourceCellInfo: cellInfo})));
const rectsByPages = splitRectsByPages(rects, options);
const pdfCellsInfoByPages = rectsByPages.map(rect => Object.assign({}, rect.sourceCellInfo, { _rect: rect }));

Reply:
Done